### DISCUSSION TITLE
介绍一期播客：中国从帝制走向现代化的时候，国家财政系统的历史包袱

### DISCUSSION THREAD

**Discussion by yibingbingwu** [RoadSideUniversity/ClubHouse2024#43] [General]
来源：

- 来自苹果博客，[链接](https://podcasts.apple.com/us/podcast/%E5%BF%BD%E5%B7%A6%E5%BF%BD%E5%8F%B3/id1493503146?i=1000635533829)

简介：

- 帝制时代，中国财政是经济学古典保守派的理想境界：金本位/紧货币，零杠杆财政，几乎没有国债负担。收多少花多少。
- 同时，没有成行的民间的债券市场可言。
- 横向比较英国和日本从中世纪走向近代时候财政的大环境，和他们是如何转型的
- 中国却很难转型：历史的包袱背了几千年了，习惯了

释义：

- 所谓”大分流“([The Great Divergence](https://www.amazon.com/Great-Divergence-Economy-Princeton-Classics/dp/0691217181/ref=sr_1_1?crid=IN360FOX20H0&keywords=the+great+divergence+pomeranz&qid=1701563887&sprefix=the+great+diver%2Caps%2C147&sr=8-1)) 是个历史学界的一个内部名词
- 原义是说，中国在中世纪前（18世纪？）和世界其他各地差不多。然后不知怎么就“分流”了

**Comment by ergatea** [comment]
挺有意思的一期

Bing Wu ***@***.***> 于2023年12月3日周日 05:04写道：

**Comment by Lewisrxliu** [comment]
许宏访谈二里头：关于中国上古史夏商考古断代的感想

中国上古文明是一个农耕文化不断胡化的历史
https://youtu.be/sabfdJrOIOk?si=Xh6R38XAuwjbCM0N
